core/src/main/java/com/yahoo/ycsb/RandomByteIterator.java

```diff
@@ -73,8 +73,7 @@ public byte nextByte() {
     return buf[bufOff - 1];
   }
 
-  @Override
-  public int nextBuf(byte[] buffer, int bufOffset) {
+  private int nextBuf(byte[] buffer, int bufOffset) {
```

Comment:
Keep it public please as mentioned above.

Reply:
Done.